asserts/gpgkeypairmgr.go

```diff
@@ -183,3 +223,83 @@ func (gkm *GPGKeypairManager) sign(fingerprint string, content []byte) ([]byte,
 	}
 	return out, nil
 }
+
+type gpgKeypairInfo struct {
+	pubKey      PublicKey
+	fingerprint string
+}
+
+func (gkm *GPGKeypairManager) findByName(name string) (*gpgKeypairInfo, error) {
+	stop := errors.New("stop marker")
+	var hit gpgKeypairInfo
+	match := func(privk PrivateKey, fpr string, uid string) error {
+		if uid == name {
+			hit = gpgKeypairInfo{
+				pubKey:      privk.PublicKey(),
+				fingerprint: fpr,
+			}
+			return stop
+		}
+		return nil
+	}
+	err := gkm.Walk(match)
+	if err == stop {
+		return &hit, nil
+	}
+	if err != nil {
+		return nil, err
+	}
+	return nil, fmt.Errorf("cannot find key named %q in GPG keyring", name)
+}
+
+var generateTemplate = `
+Key-Type: RSA
+Key-Length: 4096
+Subkey-Type: RSA
+Subkey-Length: 4096
+Name-Real: %s
+Preferences: SHA512
```

Comment:
I think for sanity we want to set the Creation-Date to v1FixedTimestamp, with that the gpg fingerprint of this and the one of the public key that goes into the account-key would agree,

it's not a must though